### ISSUE TITLE
Androids still "bleed" even though they don't

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#2251]
**Describe the bug**
Back in ye old days of PPG, androids bled oil. Then, it was removed, but androids still "bleed"

**Clearly describe how to reproduce the bug**
1. spawn android and gun
2. hahahahahah BZZZZZZZZZZZ
3. inspect
4. bleeding???

[exported game debug info.md](https://github.com/studio-minus/ppg-bugs-and-requests/files/9741716/exported.game.debug.info.md)

**Additional notes**
![20221009164539_1](https://user-images.githubusercontent.com/104321411/194760517-7efa42d4-dd74-40fe-961a-a71bb107c3ca.jpg)

**Comment by DindinYT37** [comment]
In my opinion, I think androids should bleed oil again - but it should be much harder to get a gunshot wound on them (which I'm pretty sure, is already the case).

**Comment by bibbybabby** [comment]
> In my opinion, I think androids should bleed oil again - but it should be much harder to get a gunshot wound on them. I'm pretty sure, that's already the case, though.

they never bleed oil

**Comment by DindinYT37** [comment]
> > In my opinion, I think androids should bleed oil again - but it should be much harder to get a gunshot wound on them. I'm pretty sure, that's already the case, though.
> 
> they never bleed oil

That's why I said they should?

**Comment by bibbybabby** [comment]
> > > In my opinion, I think androids should bleed oil again - but it should be much harder to get a gunshot wound on them. I'm pretty sure, that's already the case, though.
> > 
> > 
> > they never bleed oil
> 
> That's why I said they should?

you said that's already the case? as in they bleed

**Comment by DindinYT37** [comment]
> > > > In my opinion, I think androids should bleed oil again - but it should be much harder to get a gunshot wound on them. I'm pretty sure, that's already the case, though.
> > > 
> > > 
> > > they never bleed oil
> > 
> > 
> > That's why I said they should?
> 
> you said that's already the case? as in they bleed

No, as in it should be much harder to get a gunshot wound on them.
